src/v/redpanda/admin_server.cc

```diff
@@ -756,7 +756,10 @@ void admin_server::register_cluster_config_routes() {
       _server._routes,
       [this](std::unique_ptr<ss::httpd::request> req)
         -> ss::future<ss::json::json_return_type> {
-          if (!config::node().enable_central_config) {
+          if (
+            !config::node().enable_central_config
```

Comment:
Where a bool is expected and we hit a property without the typical (), we still hit the `operator T() const { return value();}` where T is bool.

Although I usually try and use the `()` to avoid any ambiguity, so I've changed this commit to add it back.